parm/config/gfs/config.stage_ic

```diff
@@ -15,7 +15,7 @@ export STAGE_IC_YAML_TMPL="${PARMgfs}/stage/master_gfs.yaml.j2"
 # Set ICSDIR (if not defined)
 if [[ -z "${ICSDIR}" ]] ; then
 
-  ic_ver="20240610"
+  ic_ver="20240829"
```

Comment:
Does this mean one needs updated initial conditions?

Reply:
My initial response to you was going to be "Nope, they're all staged" but now I'm realizing they are not for the CI.  This was something that we updated by hand for HR4 that we thought should get merged in, but now I'm thinking it will create more problems.  I'm going to remove this from this PR.